test/integration/contract.js

```diff
@@ -371,7 +374,12 @@ describe('Contract', function () {
     try {
       await deployed.call('main', [await contract.address()])
     } catch (e) {
-      e.message.indexOf('Invocation failed').should.not.be.equal(-1)
+      e.message.should.be.equal('Invocation failed')
+    }
+    try {
+      await deployed.call('foo')
+    } catch (e) {
+      e.message.should.be.equal('Invocation failed: "ICustomErrorMessage"')
```

Comment:
Still need to figure out how to remove `I` prefix

Reply:
maybe we can have hans change something about the compiler output here, so you won't have to spend time messing with this prefix issue ?

Reply:
yep, let's merge it as it is now